Week1/homework/databases_exercises/exercise1/index.js

```diff
@@ -0,0 +1,62 @@
+'use strict'
```

Comment:
The tables created in this file look different than what the exercise asks for?
https://github.com/SocialHackerCreteClass/databases/blob/master/Week1/MAKEME.md

That's fine though, the ones you've created are more advanced so cool.